lib/Parse/Lexer.cpp

```diff
@@ -1192,6 +1207,15 @@ unsigned Lexer::lexCharacter(const char *&CurPtr, char StopQuote,
   case '"': ++CurPtr; return '"';
   case '\'': ++CurPtr; return '\'';
   case '\\': ++CurPtr; return '\\';
+  case '\n':
+    LLVM_FALLTHROUGH;
```

Comment:
Per existing code in these files, no need for `LLVM_FALLTHROUGH;` when your case is otherwise empty.